pilot/pkg/serviceregistry/kube/controller/ambientindex.go

```diff
@@ -699,64 +761,50 @@ func (a *AmbientIndex) handlePod(oldObj, newObj any, isDelete bool, c *Controlle
 	a.mu.Lock()
 	defer a.mu.Unlock()
 	updates := sets.New[model.ConfigKey]()
-	// This is a waypoint update
```

Comment:
why can we remove this?

Reply:
this is all moved to `handleService`.  Originally there was no Service object created when the waypoint is deployed, so there used to be a list of waypoint podIPs but now we can use the Service IP.